bundle-size/test/common.test.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2022 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {sortBundleSizeItems} = require('../common');
+
+describe('bundle-size common', () => {
+  describe('sortBundleSizeItems', () => {
```

Comment:
I added a test for the utility, but I did not change tests for the summary output. 

(The current summary output only has one item per list, and I could not figure out how to add more entries)